frontend/src/service/Url.service.ts

```diff
@@ -149,9 +149,14 @@ export class UrlService {
           resolve(res.data.authMutation.createURL);
         })
         .catch(({ graphQLErrors, networkError, message }) => {
-          console.log(graphQLErrors);
+          if (!graphQLErrors || graphQLErrors.length === 0) {
+            if (networkError) {
```

Comment:
Let’s try to reduce the number of nested if statements ( cyclomatic complexity): https://www.google.com/amp/s/www.geeksforgeeks.org/cyclomatic-complexity/amp/

Reply:
Ah I see, so what we have here is a cyclomatic complexity of 4 because we branch and then do a branch inside of that branch (nested if). I have pushed out a fix that should now reduce the cyclomatic complexity to 2 if I am understanding this correctly, as there's now two if statements not nested. I put the check for network error on top, as I believe network errors should be prioritized over GraphQL errors.